pkgs/applications/video/dmlive/default.nix

```diff
@@ -13,16 +13,16 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "dmlive";
-  version = "5.3.0";
+  version = "5.3.1";
```

Comment:
The version number is not bumped upstream: https://github.com/THMonster/dmlive/blob/0a07fd1b831bc9e9d34e474284430297b63446c7/Cargo.toml#L3

Reply:
It seems to be an upstream mistake since the version of dmlive and [revda](https://github.com/THMonster/Revda) used to be the same.

That said, I am ok with keeping dmlive version unchanged.

WDYT?

Reply:
> It seems to be an upstream mistake.

Likely, anyway the version number is aesthetic.